src/main/java/task_modification_use_case/TaskModificationDataAccess.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Task Modification Data Access Class.
+ * Author: Raghav Arora
+ * Modified By: Raghav Arora
+ * Created: Nov 12, 2022
+ * Last Modified: Nov 19, 2022
+ */
+package task_modification_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessTask;
+import data_access.WeekDataAccess;
+import java.util.HashMap;
+
+public class TaskModificationDataAccess extends WeekDataAccess implements TaskModificationDsGateway {
+
+    public TaskModificationDataAccess() {}
```

Comment:
Why is this included if it is not doing anything?

Reply:
Removed the redundant constructor.